plugins/reposync.py

```diff
@@ -244,13 +244,14 @@ def _get_latest(self, query):
         module_dict = {}  # {NameStream: {Version: [modules]}}
         artifact_version = {} # {artifact: {NameStream: [Version]}}
         for module_package in module_packages:
-            artifacts = module_package.getArtifacts()
-            all_artifacts.update(artifacts)
-            module_dict.setdefault(module_package.getNameStream(), {}).setdefault(
-                module_package.getVersionNum(), []).append(module_package)
-            for artifact in artifacts:
-                artifact_version.setdefault(artifact, {}).setdefault(
-                    module_package.getNameStream(), []).append(module_package.getVersionNum())
+            if module_package.getRepoID() == repo_id:
```

Comment:
I am not sure whether it is a correct approach for reposync.

```if module_package.getRepoID() != hawkey.MODULE_FAIL_SAFE_REPO_NAME:```

Also the condition requires a comment.

Reply:
As we discussed I have dropped the reposync changes because:
- The bz [2072441](https://bugzilla.redhat.com/show_bug.cgi?id=2072441) didn't report that as an issue
- It is not 100% clear what is the correct approach (lets wait if someone reports it)